docker/README.md

```diff
@@ -10,6 +10,35 @@
 </div>
 <!-- markdownlint-enable no-inline-html line-length -->
 
+## Known Issue for FiftyOne Teams v1.6.0 and Above
+
+FiftyOne Teams v1.6 introduces the Central Authentication Service (CAS), which
+introduces an abstraction layer between FiftyOne Teams and Auth0.  This
+abstraction layer makes it possible to deploy FiftyOne Teams without using Auth0
+as an Identity Service Provider.
+
+However, metadata that used to be provided to FiftyOne Teams by Auth0 is no
+longer available; which has resulted in an incomplete set of instructions in the
+[Install FiftyOne](https://docs.voxel51.com/teams/installation.html#python-sdk)
+instructions for bash.
+
+Specifically, you will see the word `TOKEN` where your Voxel51 PyPI token used
+to appear.
+
+While Voxel51 works to address this issue, you can override the install
+instructions by setting the `FIFTYONE_APP_INSTALL_FIFTYONE_OVERRIDE` environment
+value for the `teams-app` deployment.  This can be accomplished by adding
```

Comment:
```suggestion
value for the `teams-app` service.  This can be accomplished by adding
```

Reply:
no, it's the deployment, not the service

Reply:
also, I'm definitely not willing to commit to 'one sentence per line' and '80 character limit' until there's a linter check that manages that.

nope, nope, nope

Reply:
In the compose yaml, it's `service`. /shrug

Reply:
in compose it _is_ a service, but a service in k8s is a load balancer

Reply:
...and this is the docker compose docs... 😢

Reply:
It's a good thing that I only made that change in the compose docs, huh?

Reply:
you should refresh faster